README.md

```diff
@@ -44,22 +48,22 @@ Visit the [Bus open data implementation guide](https://www.gov.uk/government/pub
 The following dependencies are required. An AWS account is also required.
 
 | Dependency                                                                                                                                  | Description                                                                                                      |
-| ------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+|---------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
 | [asdf](https://asdf-vm.com/guide/getting-started.html)                                                                                      | Runtime version manager                                                                                          |
```

Comment:
I think we should probably have a table that's only for dependencies that can't be installed with asdf, otherwise people might install things twice

Reply:
separated out into two tables to show what is installed via asdf